contributing/pull_request.md

```diff
@@ -38,16 +38,20 @@ This might sound a bit tough, but it is required for us to maintain the quality
 All code must conform to our [Style Guide](./styleguide.md), which is
 based on PSR-12.
 
-This makes certain that all submitted code is of the same format 
+This makes certain that all submitted code is of the same format
 as the existing code and ensures that the codebase will be as readable as possible.
 
 You can fix most of the coding style violations by running this command in your terminal:
 
-	composer cs-fix
+```
+> composer cs-fix
+```
 
 You can check the coding style violations:
 
-	composer cs
+```
```

Comment:
Specify `bash` formatting?

Reply:
Added `console`.

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting
https://github.com/github/linguist/blob/d7799da826e01acdb8f84694d33116dccaabe9c2/lib/linguist/languages.yml#L6063

Reply:
Wasn't aware of "ShellSession", very cool.